app/controllers/instructor/photos_controller.rb

```diff
@@ -17,7 +17,7 @@ def current_course
   end
 
   def photo_params
-    params.require(:photo).permit(:photo)
+    params.require(:photo).permit(:image)
```

Comment:
I changed the field from `photo` to `image`.  The `image` refers to the mounted uploader in carrierwave, and `photo` is the database table structure for it.